notebooks/instructlab-knowledge/instructlab-knowledge.ipynb

```diff
@@ -144,21 +152,131 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 14,
    "id": "a200039c-b8b2-4087-88ba-7bfb0e393cc9",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Path of JSON output is: /home/alina/dev/FNF/examples/notebooks/instructlab-knowledge/workspaces/default/conversion/BofA_InterestChecking_en_ADA.json\n"
+     ]
+    }
+   ],
    "source": [
     "import json\n",
     "\n",
+    "converted_json_paths = []\n",
+    "\n",
     "for file in files:\n",
     "    doc = doc_converter.convert(source=file).document\n",
     "    doc_dict = doc.export_to_dict()\n",
     "\n",
     "    json_output_path = CONVERSION_OUTPUT_DIR / f\"{file.stem}.json\"\n",
     "    with open(json_output_path, \"w\") as f:\n",
     "        json.dump(doc_dict, f)\n",
-    "        print(f\"Path of JSON output is: {Path(json_output_path).resolve()}\")"
+    "        print(f\"Path of JSON output is: {Path(json_output_path).resolve()}\")\n",
+    "    converted_json_paths.append(Path(json_output_path).resolve())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "40710019-7ec9-414e-ad72-1ba672cf5fc2",
+   "metadata": {},
+   "source": [
+    "## Post-Conversion: Illuminator Analysis"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2572e2d0-94dc-4ca0-b032-3978af26c9c9",
+   "metadata": {},
+   "source": [
+    "This step guides you through analyzing docling-converted documents for problematic table structures, specifically merged table cells, using core functions from the Illuminator codebase."
```

Comment:
I think that this text block is an opportunity to explain what your tool does, and why they should optionally use it. I think phrasing this to tell that story might be clearer. Here is an example:

```
Document conversion isn't always perfect. Data can get distorted or corrupted, which may result in lower quality training outcomes. It is optional, but strongly recommended, that you review your converted data. This example uses the Illuminator tool to scan converted documents for common issues.
```

I would highly recommend working with @kelbrown20  on this

Reply:
Sounds great to me

Reply:
+1 I like it!